flocker/node/agents/blockdevice.py

```diff
@@ -198,6 +205,22 @@ def __init__(self, device):
     u"A block-device-backed dataset is being created.",
 )
 
+FLOCKER_1_0_0_FS_MOUNTED = MessageType(
+    u"agent:blockdevice:1.0.0:mounted:fs",
+    [BLOCK_DEVICE_ID, BLOCK_DEVICE_PATH],
+    u"A filesystem created by Flocker 1.0.0 was encountered and was mounted "
+    u"so it could not be upgraded.",
+)
+
+FLOCKER_1_0_0_FS_UPGRADE = ActionType(
+    u"agent:blockdevice:1.0.0:upgrade:fs",
+    [BLOCK_DEVICE_ID, DATASET_ID],
+    fields(old_fs_uuid=unicode, tune2fs_output=unicode),
+    u"A filesystem created by Flocker 1.0.0 was encountered and its "
+    u"filesystem UUID was changed to match the dataset id it is meant to "
+    u"represent.",
```

Comment:
But this message doesn't actually show up in the logs, right? So maybe ignore that.